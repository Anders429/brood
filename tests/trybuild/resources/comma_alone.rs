use brood::resources;

fn main() {
    let resources = resources!(,);
}
