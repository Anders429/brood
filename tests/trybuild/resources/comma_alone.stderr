error: no rules expected the token `,`
 --> tests/trybuild/resources/comma_alone.rs:4:32
  |
4 |     let resources = resources!(,);
  |                                ^ no rules expected this token in macro call
